packages/remote-config/test/remote_config.test.ts

```diff
@@ -58,7 +58,7 @@ describe('RemoteConfig', () => {
 
   let getActiveConfigStub: sinon.SinonStub;
   let loggerDebugSpy: sinon.SinonSpy;
-  let loggerLogLevelSpy: sinon.SinonSpy;
+  let loggerLogLevelSpy: any;
```

Comment:
Are we not applying eslint in test files?

Reply:
`no-explicit-any` is always off in test files: https://github.com/firebase/firebase-js-sdk/blob/e1173544b2f63691e887622b827bcc819e743293/config/.eslintrc.js#L15

which was added in https://github.com/firebase/firebase-js-sdk/pull/1845

Which I definitely like because setting this type to `any` seemed like it was the only way I could make this spy work.